awscli/customizations/scalarparse.py

```diff
@@ -36,13 +39,29 @@ def identity(x):
     return x
 
 
+def iso_format(value):
+    return parse_timestamp(value).isoformat()
+
+
+def choose_default_parsers(timestamp_format):
```

Comment:
Yup looks like I completely misread that. I'll reshuffle things to make more sense.